packages/components/src/ecosystem_bar/index.tsx

```diff
@@ -1,23 +1,39 @@
 import clsx from 'clsx';
-import PropTypes from 'prop-types';
 import { useRef } from 'react';
 import { useOnceSwipeTracking, useScrolled } from '@/src/ecosystem_bar/hooks';
 import styles from './index.module.scss';
 
-const EcosystemBar = ({ items, onFirstSwipe, onItemClick }) => {
-  const navRef = useRef(null);
+type EcosystemBarItem = {
+  key: string,
+  link: string,
+  isActive: boolean,
+  event: string,
+  text: {
+    mobile: string,
+    others: string,
+  },
+};
+
+export type EcosystemBarProps = {
+  items: EcosystemBarItem[];
+  onFirstSwipe: () => void;
+  onItemClick: (event: string) => void
+};
```

Comment:
Just for my understanding: why do we export the EcosystemBarProps here?

Reply:
Good question, I was just doing it like the other components with the assumption so that it can be found when using the component in our apps. but then again I am new to this too :-) @peterhass  ?

Reply:
My assumption would be: I think there is no need in this case to export this props because they are bound to EcosystemBar and not really needed somewhere else. 
As I understood it: for more general types that might be needed somewhere else we can export those in a index.d.ts file on an upper root level [like here](https://github.com/good-hood-gmbh/nebenan-frontend/blob/fbe013e0d506f7a29ba0f9de965e3a1a52b3209f/client/types.tsx#L4) e.g.

Reply:
True. But I will instead export the EcosystemBarItem type since this will be used outside in the nebenan-frontend https://github.com/good-hood-gmbh/nebenan-frontend/blob/e0c586260179963722a50ed5f7a6d89dbd2769cd/client/components/layout/ecosystem_nav_bar/utils.jsx#L4